src/main/java/ru/rosatom/documentflow/dto/DocProcessCommentDto.java

```diff
@@ -0,0 +1,20 @@
+package ru.rosatom.documentflow.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+@Schema(description = "Комментарий")
+public class DocProcessCommentDto {
+    @Schema(description = "Текст комментария")
+    private String textComment;
+    @Schema(description = "Автор комментария")
+    private UserReplyDto authorComment;
+    @Schema(description = "Дата создания комментария")
+    private LocalDateTime date;
```

Comment:
date чего? Понятно что в Schema всё написано, но Schema отображается в сваггере, при работе за пределами этого класса сложно понять что за дата 
```suggestion
    @Schema(description = "Дата создания комментария")
    private LocalDateTime createdAt;
```